vm/inc/ubpf.h

```diff
@@ -134,7 +134,8 @@ int ubpf_translate(struct ubpf_vm *vm, uint8_t *buffer, size_t *size, char **err
  *  ubpf_get_inst_cnt(): Gets the instruction count struct.
  */
 
-void ubpf_enable_inst_cnt(struct ubpf_vm *vm);
+void ubpf_set_inst_cnt_enable(struct ubpf_vm *vm, bool enable);
+bool ubpf_get_inst_cnt_enable(struct ubpf_vm *vm);
```

Comment:
Doesn't ubpf_get_inst_cnt() return this? Why do we need a separate function for it. Also, it should probably be in the previous patch.